scala-package/examples/pom.xml

```diff
@@ -14,6 +14,24 @@
   <name>MXNet Scala Package - Examples</name>
 
   <profiles>
+    <profile>
+      <id>osx-x86_64-cpu</id>
```

Comment:
Is the default behaviour windows or why is it not defined here?

Reply:
there is no support for windows in the current Scala-package and there are no plans to add with this change. This has to be taken up as a separate task for MXNet-Scala binding

Reply:
Ah I wasn't aware of that, thanks for elaborating